dev/docker/README.md

```diff
@@ -0,0 +1,86 @@
+Apache REEF (incubator) Docker Tests
+====================================
+
+[Apache REEF (incubator)](http://reef.incubator.apache.org/) is 
+an approach to simplify and unify the lower layers of big data systems
+on modern resource managers. This project maintains docker images
+to simulate the target underlying systems as similar as possible.
+
+Docker-based Test Cluster
+-------------------------
+
+| OS           | Java            | Hadoop Version            | Description   | alias     |
+|--------------|-----------------|---------------------------|---------------|-----------|
+| Ubuntu 12.04 | Oracle JDK 7u80 | HDP 2.1.15 (Hadoop 2.4.0) | HDInsight 3.1 | hdi3.1    |
+| Ubuntu 12.04 | Oracle JDK 7u80 | HDP 2.2.7  (Hadoop 2.6.0) | HDInsight 3.2 | hdi3.2    |
+| Ubuntu 12.04 | Oracle JDK 7u80 | HDP 2.2.8  (Hadoop 2.6.0) |               | hdp2.2    |
```

Comment:
Thank you, I forgot to trigger it. I trigger the build now. Maybe, it'll show up in 1 hour.
By the way, please note that the current prebuilt images in DockerHub are based on my repository instead of Apache REEF. After committing, I should change the GitHub location.

If you want to see the result right now, you can build it locally. 
```
docker build -t reefrt/hdp2.2 ubuntu12.04-jdk7-hdp2.2.8
```

I thought the audience of README.md is not reviewer. So, there is some mismatch. In any way, sorry for that.